cl/tests/test_feeds.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+"""
+Functional testing of courtlistener RSS feeds
+"""
+from django.conf import settings
+from django.test.utils import override_settings
+from cl.search.models import Court
+from cl.tests.base import BaseSeleniumTest
+import feedparser
+import os
+
+
+@override_settings(
+    MEDIA_ROOT = os.path.join(settings.INSTALL_ROOT, 'cl/assets/media/test/')
+)
+class FeedsFunctionalTest(BaseSeleniumTest):
+    """Tests the Feeds page and functionality"""
+
+    fixtures = ['test_court.json', 'judge_judy.json', 'functest_opinions.json',
+        'functest_audio.json']
+
+    def test_can_get_to_feeds_from_homepage(self):
+        """Can we get to the feeds/podcasts page from the homepage?"""
+        self.browser.get(self.server_url)
+        link = self.browser.find_element_by_link_text('Feeds & Podcasts')
+        link.click()
+
+        self.assertIn('Feeds & Podcasts', self.browser.title)
+        self.assertIn('/feeds', self.browser.current_url)
+        self.assert_text_in_body('Feeds & Podcasts')
+
+    def test_feeds_page_shows_jurisdiction_links(self):
+        """
+        Does the feeds page show all the proper links for each jurisdiction?
+        """
+        courts = Court.objects.all()
+        self.browser.get('%s/feeds' % (self.server_url,))
```

Comment:
This should use [reverse][1] instead. Ideally we shouldn't ever hard code URLs anywhere other than the urls.py file. This one, for example can be replaced with:

    '%s%s' % (self.server_url, reverse('feed_info'))

You can get a list of all the URLs and their names by using something like:

    manage.py show_urls | grep feeds

(Though this is only available if you have `django-extensions` installed.)


[1]: https://docs.djangoproject.com/en/1.8/ref/urlresolvers/#reverse